src/main/java/uk/gov/ons/ctp/response/casesvc/domain/model/CaseEvent.java

```diff
@@ -66,6 +56,6 @@ public class CaseEvent implements Serializable {
   private String subCategory;
 
   @Column(name = "metadata")
-  @Type(type = "jsonb")
+  @JdbcTypeCode(SqlTypes.JSON)
```

Comment:
Previous hibernate `@Type` for jsonb declarations has been changed to this new approach